features/exploredash/src/main/java/org/dash/wallet/features/exploredash/AppExploreDatabase.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2022 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.features.exploredash;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.room.Database;
+import androidx.room.Room;
+import androidx.room.RoomDatabase;
+import androidx.room.TypeConverters;
+import androidx.sqlite.db.SupportSQLiteDatabase;
+
+import org.dash.wallet.common.Configuration;
+import org.dash.wallet.common.data.RoomConverters;
+import org.dash.wallet.features.exploredash.data.AtmDao;
+import org.dash.wallet.features.exploredash.data.MerchantDao;
+import org.dash.wallet.features.exploredash.data.model.Atm;
+import org.dash.wallet.features.exploredash.data.model.AtmFTS;
+import org.dash.wallet.features.exploredash.data.model.Merchant;
+import org.dash.wallet.features.exploredash.data.model.MerchantFTS;
+import org.dash.wallet.features.exploredash.repository.ExploreRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+@Database(entities = {
+        Merchant.class,
+        MerchantFTS.class,
+        Atm.class,
+        AtmFTS.class
+}, version = 1)
+@TypeConverters({RoomConverters.class})
+public abstract class AppExploreDatabase extends RoomDatabase {
```

Comment:
This file is moved to the Explore module and is largely the same except for the mentioned below changes.